src/modules/75-cf/components/FlagArchiving/__tests__/CannotArchiveWarning.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { render, RenderResult, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { TestWrapper } from '@common/utils/testUtils'
+import mockFeature from '@cf/utils/testData/data/mockFeature'
+import type { Feature } from 'services/cf'
+import CannotArchiveWarning, { CannotArchiveWarningProps } from '../CannotArchiveWarning'
+
+const renderComponent = (props: Partial<CannotArchiveWarningProps> = {}): RenderResult => {
+  return render(
+    <TestWrapper
+      path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
+      pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
+    >
+      <CannotArchiveWarning flagData={mockFeature} {...props} />
+    </TestWrapper>
+  )
+}
+describe('CannotArchiveWarning', () => {
+  test('it should render the list of flags the current flag is dependent on and take the user to the detail page of the prereq flag', async () => {
+    renderComponent()
+
+    expect(screen.getByText('cf.featureFlags.archiving.cannotArchive:')).toBeInTheDocument()
+    expect(screen.getByText('cf.featureFlags.archiving.removeFlag')).toBeInTheDocument()
+
+    mockFeature.prerequisites!.forEach(prereq =>
+      expect(screen.getByRole('link', { name: prereq.feature })).toBeInTheDocument()
+    )
+
+    userEvent.click(screen.getByRole('link', { name: mockFeature.prerequisites![0].feature }))
+
+    // directing to the feature flag detail page
+    await waitFor(() =>
+      expect(screen.getByTestId('location')).toHaveTextContent(
+        '/account/dummy/cf/orgs/dummy/projects/dummy/feature-flags/Test_Paging_Flag'
+      )
+    )
```

Comment:
Nothing wrong here, I just (for some reason) really like using `findBy` for these kinda things.
```suggestion
    expect(await screen.findByTestId('location')).toHaveTextContent(
      '/account/dummy/cf/orgs/dummy/projects/dummy/feature-flags/Test_Paging_Flag'
    )
```